pokeapp/src/components/FilterBar.js

```diff
@@ -1,22 +1,45 @@
-import React, {useState, useEffect} from "react";
+import React, { useState, useEffect } from "react";
 
 //Enumerator Object
 const SortTypes = {
-A_TO_Z: 1,
-Z_TO_A: 2,
-ID_HIGH_TO_LOW: 3, 
-ID_LOW_TO_HIGH: 4,
-VALUE_HIGH_TO_LOW: 5,
-VALUE_LOW_TO_HIGH: 6, 
-}
-
-const FilterBar = ({ allPokemon, setAllPokemonFiltered }) => {
-
-const [sortType, setSortType] = useState(SortTypes.ID_LOW_TO_HIGH);
-const [type, setType] = useState(false);
+  A_TO_Z: 1,
+  Z_TO_A: 2,
+  ID_HIGH_TO_LOW: 3,
+  ID_LOW_TO_HIGH: 4,
+  VALUE_HIGH_TO_LOW: 5,
+  VALUE_LOW_TO_HIGH: 6,
+};
+//Enumerator by Element
+const ElementTypes = {
+  BUG: "bug",
+  DARK: "dark",
+  DRAGON: "dragon",
+  ELECTRIC: "electric",
+  FAIRY: "fairy",
+  FIGHTING: "fighting",
+  FIRE: "fire",
+  FLYING: "flying",
+  GRASS: "grass",
+  GHOST: "ghost",
+  GROUND: "ground",
+  ICE: "ice",
+  NORMAL: "normal",
+  POISON: "poison",
+  PSYCHIC: "psychic",
+  ROCK: "rock",
+  STEEL: "steel",
+  WATER: "water",
+};
```

Comment:
We should extract these out to an `enums.js` file.

Reply:
Wow, that code snippet got borked.  I mean to say that this file is big, and we should extract out some of the less directly related things.